opm/parser/share/keywords/000_Eclipse100/W/WCONPROD

```diff
@@ -7,7 +7,7 @@
        {"name" : "GRAT"         , "value_type" : "DOUBLE" , "default" : 0.0 , "dimension" : "GasSurfaceVolume/Time"},
        {"name" : "LRAT"         , "value_type" : "DOUBLE" , "default" : 0.0 , "dimension" : "LiquidSurfaceVolume/Time"},
        {"name" : "RESV"         , "value_type" : "DOUBLE" , "default" : 0.0 , "dimension" : "LiquidSurfaceVolume/Time"},
-       {"name" : "BHP"          , "value_type" : "DOUBLE" , "default" : 0.0 , "dimension" : "Pressure"},
+       {"name" : "BHP"          , "value_type" : "DOUBLE" , "default" : 1.01325 , "dimension" : "Pressure"},
```

Comment:
> What units should default values use in the keyword?

The defaults should use metric units - when resolving defaults two different unit system are in action, the unit system of the deck, and the default unit system (which is metric).

> Is there a way to use the unit constants from the C++ code here somehow? (I guess not.)

Yes there is - see eg: https://github.com/OPM/opm-parser/blob/master/opm/parser/eclipse/EclipseState/Tables/Tabdims.hpp#L42

> Is there a way to put comments in these files?

Well - as you JSON itself does not support comments, so there is no first class support for comments. However - no one is stopping you from something like:
```json
{"name" : "BHP", 
 "value_type" : "DOUBLE" , 
 "default" : 1.01325 , 
 "dimension" : "Pressure",
 "comment"  : "The default corresponds to ...."}
```
however that will not be internalized in any way.